code/modules/projectiles/guns/projectile/siren.dm

```diff
@@ -8,12 +8,13 @@
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guninhands_left.dmi', "right_hand" = 'icons/mob/in-hand/right/guninhands_right.dmi')
 	recoil = 1
 	slot_flags = SLOT_BELT
-	flags = FPRINT | NOREACT
+	flags = FPRINT | NOREACT | OPENCONTAINER
```

Comment:
Oh god it has `NOREACT`
I can't wait for a pacidsmoke gun

(That's actually not possible even though this has `NOREACT` because of pacid's custom touch reaction preventing it from entering a mob's system on splash, but it's fun to think about)